contracts/ocr2/src/state.rs

```diff
@@ -35,12 +35,17 @@ pub struct Validator {
     pub gas_limit: u64,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq, JsonSchema)]
 pub struct Billing {
     /// Should match https://fcd.terra.dev/v1/txs/gas_prices
     /// NOTE: needs to be scaled to the same amount of decimals places as LINK token
     pub recommended_gas_price: u64,
```

Comment:
Suggestion by @archseer:

> for recommended_gas_price: the API returns uLUNA but we want to turn that into a u128 with 18 decimal places
> (uluna = micro luna = luna * 10⁻⁶)
>
> there wasn't an easy way to construct a decimal and say "this only has 6 decimal places" because it hardcodes to 18 decimals https://github.com/smartcontractkit/chainlink-terra/blob/9c529e3c016e23234cdbde8b4d9c7952ebe2912f/contracts/ocr2/src/decimal.rs#L30
>
> well there's https://github.com/smartcontractkit/chainlink-terra/blob/9c529e3c016e23234cdbde8b4d9c7952ebe2912f/contracts/ocr2/src/decimal.rs#L76-L79 so we could technically support uluna

Reply:
GH Issue:

- https://github.com/smartcontractkit/chainlink-terra/issues/59